### DISCUSSION TITLE
Rendering and performance issue on Android

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#17936] [General]
Below are the screenshots of the page showing controls not rendering on Android:

WinAppSDK
![image](https://github.com/user-attachments/assets/41f73e71-e08e-49f5-a52c-3525acd0f47d)
Android
![image](https://github.com/user-attachments/assets/b60624c1-395a-43f3-af40-757406427e85)

Also Click on "**View Next Page**" at bottom, you can see same sets of controls takes way longer to render.
Here is the repro:
[TestProj.zip](https://github.com/user-attachments/files/16606846/TestProj.zip)

**Comment by Youssef1313** [comment]
The behavior I'm observing on Android appears to be different than what you're observing :eyes:

![image](https://github.com/user-attachments/assets/2b5cfab3-ca08-4c69-b06a-0ad0628d3503)

**Reply by WenHong-Lau** [reply]
Hi @Youssef1313, were all the controls rendered correctly on the project page on your end? Also, did you experience any significant delay in rendering the records on the second page?

**Comment by WenHong-Lau** [comment]
Hi, any update on this?
